index.html

```diff
@@ -214,5 +214,52 @@ <h1>About Me</h1>
         </div>
       </section>
     </main>
+    <footer>
+      <section class="get-in-touch">
+        <p>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </p>
+      </section>
+      <form action="https://formspree.io/f/meqdgwge" method="POST">
+        <div class="form-group">
+          <input
+            type="text"
+            name="name"
+            placeholder="Full Name"
+            maxlength="30"
+            required
+          />
+          <input type="email" name="email" placeholder="Email" required />
+          <textarea
+            cols="33"
+            rows="3"
+            name="message"
+            placeholder="Enter text here"
+            required
+          ></textarea>
```

Comment:
- [ ] Kindly make sure the Message input has maxlength="500" attribute as it was required in the project so that it will imitate the text and it will reduce the possibility to input the wrong information  :+1: